### DISCUSSION TITLE
Blacklist/Whitelist not "PC" and I've been asked to maintain a separate fork. Thoughts on Allow/Deny lists? I know... smh

### DISCUSSION THREAD

**Discussion by Fimeg** [Romanitho/Winget-AutoUpdate#267] [General]
Basically summed it up in the title, thought this would be an amazing thing to implement at company, company loves it, aside from verbiage. While simple enough to maintain a fork and rather search/replace - thought it couldn't hurt to ask.. any thoughts (no way, or meh, okay can do?)

**Comment by KnifMelti** [comment]
Hi!
I don't quite follow you...
...do you mean Politically Correct with "PC"?
What do you want to accomplish.?

**Reply by Fimeg** [reply]
Yes, they are hoping the change the verbiage to 'more inclusive and politically correct terms', e.g., changing blacklist/whitelist to deny/allow lists.

**Reply by KnifMelti** [reply]
🧐 He, this I'll send over to @Romanitho to decide...
...I'm 60 years old and I don't put that meaning in the words black/white list.

**Reply by Romanitho** [reply]
to be honest, I don't really care ^^ feel free to PR and implement changes regarding "words". I just want something that works 😇

**Reply by KnifMelti** [reply]
Can't help myself...
...what do they call their whiteboards?

**Comment by KnifMelti** [comment]
Ah, marker board https://ell.stackexchange.com/questions/111996/is-there-a-specific-name-for-the-blackboard-when-its-white#comment225831_112001
